mango_base/mango/dashboards/time_series_utils/ui_components.py

```diff
@@ -0,0 +1,378 @@
+import streamlit as st
+import pandas as pd
```

Comment:
Review if this script .py must be here or in the time-series-dashboard branch.
@antoniogonzalezsuarez